src/components/ui/Tags.svelte

```diff
@@ -0,0 +1,189 @@
+    if (
+      event.target instanceof Element &&
+      container &&
+      !container.contains(event.target) &&
+      !autocompleteElement[0].contains(event.target)
+    ) {
+      closeAutocomplete();
+    }
+  }
+
+  onMount(() => {
+    try {
+      document.addEventListener('click', attemptCloseAutocomplete);
+      document.addEventListener('focusin', attemptCloseAutocomplete);
+
+      // Disable HTML autocomplete for this field
+      input = container.getElementsByTagName('input')[0];
+      input.autocomplete = 'off';
+    } catch (err) {
+      console.error(err);
+    }
+  });
+
+  onDestroy(() => {
+    // Ensure value is cleared;
+    input.value = '';
+
+    // Close autocomplete
+    closeAutocomplete();
+    document.removeEventListener('click', attemptCloseAutocomplete);
+    document.removeEventListener('focusin', attemptCloseAutocomplete);
+  });
+</script>
+
+<div bind:this={container}>
+  <Tags
+    on:tags={handleTags}
+    allowPaste={true}
+    allowDrop={true}
+    onlyUnique={true}
+    autoComplete={autocompleteValues}
+    disable={disabled}
+    {placeholder}
+    {tags}
+    {name}
+  />
+</div>
+
+<style>
+  :global(.svelte-tags-input-layout) {
+    background-color: var(--st-input-background-color) !important;
+    border: var(--st-input-border) !important;
+    border-radius: 4px !important;
+    color: var(--st-input-color) !important;
+    padding: 3px 4px !important;
+    display: flex;
+    gap: 4px;
+  }
+  :global(.svelte-tags-input) {
+    margin: 0 !important;
+    padding: 0 !important;
+    padding-left: 2px !important;
+    background: inherit;
+    font-family: var(--st-input-font-family) !important;
+    font-size: var(--st-input-font-size) !important;
+    font-weight: var(--st-input-font-weight) !important;
+    line-height: var(--st-input-line-height) !important;
+    letter-spacing: var(--st-input-letter-spacing) !important;
+  }
+  :global(.svelte-tags-input-tag) {
+    margin-top: 0 !important;
+    white-space: normal !important;
+    margin-right: 0 !important;
+    padding: 0 !important;
+    background-color: #e0e8f9 !important;
+    border-radius: 2px !important;
+    gap: 2px;
+    padding: 0px 0px 0px 2px !important;
+    font-family: var(--st-typography-body-font-family) !important;
+    color: var(--st-typography-body-color) !important;
+    font-weight: var(--st-typography-body-font-weight) !important;
+    font-size: var(--st-typography-body-font-size) !important;
+    line-height: var(--st-typography-body-line-height) !important;
+    letter-spacing: var(--st-typography-body-letter-spacing) !important;
+    cursor: default !important;
+  }
+  :global(.svelte-tags-input-tag-remove) {
+    font-weight: 600;
+    opacity: 0.5;
+    padding: 0px 2px;
+    border-top-right-radius: 2px;
+    border-bottom-right-radius: 2px;
+    width: 16px;
+    display: flex;
+    justify-content: center;
+    cursor: pointer;
+  }
```

Comment:
Can we make sure classes separated by a newline? I'll see about adding a styleint rule for this